src/mcp/server/fastmcp/server.py

```diff
@@ -1206,6 +1206,7 @@ async def log(
         message: str,
         *,
         logger_name: str | None = None,
+        **extra: Any,
```

Comment:
If we want to create this extra, let's not make it a kwargs.


```suggestion
        extra: dict[str, Any],
```